libs/javascript/@local/eslint-config/legacy-base-eslintrc-to-refactor.cjs

```diff
@@ -28,71 +28,22 @@ module.exports = {
       "error",
       {
         selector:
-          "TSTypeReference[typeName.name=/^(DirectEditorProps|EditorProps|EditorView|EditorState|NodeView|ProsemirrorNode|Command|Transaction|Plugin|PluginKey)$/]:not([typeParameters])",
+          "TSTypeReference[typeName.name=/^(Plugin|PluginKey)$/]:not([typeParameters])",
```

Comment:
Plugins are still generic (one arg instead of two)

Reply:
The argument it has is required, though – it doesn't default to any, so you don't need this lint rule